### PR TITLE
Tsegb

### PR DESCRIPTION
**Pull request by marijana777**
Added Segment B, changes in  dark_to_exposure, aperture range, etc

### CODE REVIEW

File: cosredux/darks.py [hunk 1]
```diff
@@ -420,10 +532,10 @@ def find_darks(darksfld, scifile, segm, hvlvl, ndays=90):
     darkfiles = glob.glob(darksfld + '*' + segm + '.fits')
 
     dlist = []
-    for ifile in darkfiles: # in np.arange(n):
+    for ifile in darkfiles:
         hdu = fits.open(ifile)
         head1 = hdu[1].header
-        ###d dts[i]=head1['EXPTIME']  # all are 1330
+        # dts[i]=head1['EXPTIME']  # all are 1330
```
Comment (profxj) [inline cosredux/darks.py]: Should you remove this?

File: cosredux/science.py [hunk 2]
```diff
@@ -208,3 +222,30 @@ def coadd_exposures(x1d_files, segm, outfile, bin=None):
...
+def combinespectfiles(spfile_a, spfile_b, file_ab):
+    """ Coadd two spectra, and write output in a file
+
+    Parameters
+    ----------
+    spfile_a : str
+    spfile_b : str
+       files with spectra
+    file_ab : str
+       output file with combined spectra
```
Comment (profxj) [inline cosredux/science.py]: FITS ?

I see.  It probably depends on the extension of the filename.
You should probably note this.

File: cosredux/trace.py [hunk 3]
```diff
@@ -73,15 +68,19 @@ def crude_histogram(yfull, ymin=300, ymax=700, ytl=550, pk_window=4.5, verbose=F
...
+    yfull : ndarray
+    y_guess : float, int
+      peak guess
+    pk_window : float, int, optional
+    per_lim : tuple, optional
+      percentiles limits
 
     Returns
     -------
+    pk : float
```
Comment (profxj) [inline cosredux/trace.py:83]: explain what this is

**Review by profxj — APPROVED** [review]
Looks great.  Make the few modifications and merge.

**Comment by marijana777** [comment]
Will merge

**Pull request merged**